components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -146,10 +171,18 @@ export class TeamDBImpl implements TeamDB {
             );
         }
 
+        let slug = slugify(name, { lower: true });
         const teamRepo = await this.getTeamRepo();
+
+        const existingTeam = await teamRepo.findOne({ slug, deleted: false, markedDeleted: false });
+        if (!!existingTeam) {
+            slug = slug + "-" + randomBytes(10).toString("hex");
```

Comment:
**suggestion**: If possible, I'd show only 8 characters here to make this more readable. Thoughts?

| BEFORE | AFTER |
|-|-|
| <img width="472" alt="slug-before" src="https://user-images.githubusercontent.com/120486/226886640-1ee51b43-696f-48d5-af47-543df936839b.png"> | <img width="472" alt="slug-after" src="https://user-images.githubusercontent.com/120486/226886644-001f6cd0-cd82-4d99-a9a7-672ae235a480.png"> |

Reply:
changing to 8. fine with me.